assets/bingo/lang/pl_pl.json

```
@@ -0,0 +1,411 @@
+	"bingo.lobby.settings.player.save.new.success": "Nowa konfiguracja “%s” zapisana pomyślnie",
+	"bingo.lobby.settings.player.save.overwrite.success": "Pomyślnie nadpisano konfigurację “%s” nowymi ustawieniami.",
+
+	"bingo.lobby.settings.player.load.sign.line1": "Gracz",
+	"bingo.lobby.settings.player.load.sign.line2": "",
+	"bingo.lobby.settings.player.load.sign.line3": "Wybór/Podgląd",
+	"bingo.lobby.settings.player.load.sign.line4": "Konfiguracji",
+	"bingo.lobby.settings.player.load.title": "Załaduj Konfigurację Gracza",
+	"bingo.lobby.settings.player.load.description": "Konfiguracje do załadowania:",
+	"bingo.lobby.settings.player.load.new.success": "Konfiguracja “%s” załadowana pomyślnie",
+
+	"bingo.lobby.settings.player.configuration.empty": "Pusta",
+	"bingo.lobby.settings.player.configuration.night_vision": "Noktowizja",
+	"bingo.lobby.settings.player.configuration.boats": "Noktowizja + Łodzie Łodzie Łodzie",
+	"bingo.lobby.settings.player.configuration.elytra": "Noktowizja + Elytry",
+	"bingo.lobby.settings.player.configuration.frostwalker": "Noktowizja + Mroźny Piechur",
+	"bingo.lobby.settings.player.configuration.underwater": "Noktowizja + Oddychanie pod Wodą + Głębinowy Wędrowiec",
+
+	"bingo.lobby.settings.preferences.sign.line1": "",
+	"bingo.lobby.settings.preferences.sign.line2": "Preferencje",
+	"bingo.lobby.settings.preferences.sign.line3": "Gracza",
+	"bingo.lobby.settings.preferences.sign.line4": "",
+
+	"bingo.lobby.settings.pregen_behavior.sign.line1": "Wstępne",
+	"bingo.lobby.settings.pregen_behavior.sign.line2": "generowanie",
+	"bingo.lobby.settings.pregen_behavior.sign.line3": "terenu",
+	"bingo.lobby.settings.pregen_behavior.sign.on": "Przy starcie gry",
+	"bingo.lobby.settings.pregen_behavior.sign.off": "Przy los. karty",
+	"bingo.lobby.settings.pregen_behavior.info.on_game_start.prefix": "Przy starcie gry: ",
+	"bingo.lobby.settings.pregen_behavior.info.on_game_start": "Teren jest wstępnie generowany wokół spawnu, gdy tylko klikniesz na tabliczkę “Start Gry”. Powoduje to dodatkowy czas oczekiwania po rozpoczęciu gry.",
+	"bingo.lobby.settings.pregen_behavior.info.on_card_generation.prefix": "Przy losowaniu karty: ",
+	"bingo.lobby.settings.pregen_behavior.info.on_card_generation": "Generowanie wstępne rozpocznie się automatycznie 5 sekund po wylosowaniu nowej karty. Może to powodować opóźnienia w lobby, przez co niektóre funkcje w lobby będą reagować powoli, a w niektórych przypadkach nie będą działać w ogóle.",
+	"bingo.lobby.settings.pregen_behavior.turn_on": "Przełącz na “Przy starcie gry”",
+	"bingo.lobby.settings.pregen_behavior.turn_off": "Przełącz na “Przy losowaniu karty”",
```

```suggestion
	"bingo.lobby.settings.pregen_behavior.info.on_game_start": "Teren jest wstępnie generowany wokół spawnu, gdy tylko klikniesz na tabliczkę „Start Gry”. Powoduje to dodatkowy czas oczekiwania po rozpoczęciu gry.",
	"bingo.lobby.settings.pregen_behavior.info.on_card_generation.prefix": "Przy losowaniu karty: ",
	"bingo.lobby.settings.pregen_behavior.info.on_card_generation": "Generowanie wstępne rozpocznie się automatycznie 5 sekund po wylosowaniu nowej karty. Może to powodować opóźnienia w lobby, przez co niektóre funkcje w lobby będą reagować powoli, a w niektórych przypadkach nie będą działać w ogóle.",
	"bingo.lobby.settings.pregen_behavior.turn_on": "Przełącz na „Przy starcie gry”",
	"bingo.lobby.settings.pregen_behavior.turn_off": "Przełącz na „Przy losowaniu karty”",
```
Quotation mark styling